build/android.mk

```diff
@@ -27,6 +27,11 @@ DX = $(ANDROID_BUILD_TOOLS_DIR)/dx
 
 ANDROID_LIB_NAMES = xcsoar
 
+ifeq ($(TARGET)$(ANDROID_LEGACY),ANDROIDy)
```

Comment:
Why check `$(TARGET)` here? Setting `ANDROID_LEGACY=y` on a target other than Android is illegal, isn't it?

Reply:
Removed.